app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -34,10 +34,11 @@ HOMEPAGE="https://github.com/coreos/rkt"
 
 LICENSE="Apache-2.0"
 SLOT="0"
-IUSE="doc examples +rkt_stage1_coreos +rkt_stage1_fly rkt_stage1_host rkt_stage1_src +actool"
+IUSE="doc examples +rkt_stage1_coreos +rkt_stage1_fly rkt_stage1_host rkt_stage1_src +actool tpm"
 REQUIRED_USE="|| ( rkt_stage1_coreos rkt_stage1_fly rkt_stage1_host rkt_stage1_src )"
 
-COMMON_DEPEND="sys-apps/acl"
+COMMON_DEPEND="sys-apps/acl
+		tpm? ( app-crypt/trousers )"
```

Comment:
i assume a build dep since it is compiled and linked in at build time but i don't really know how deps in portage work.